ui/components/app/transaction-list/transaction-list.component.js

```diff
@@ -719,11 +719,7 @@ export default function TransactionList({
                 )}
               </Box>
             ) : (
-              <Box className="transaction-list__empty">
-                <Box className="transaction-list__empty-text">
-                  {t('noTransactions')}
-                </Box>
-              </Box>
+              <TransactionActivityEmptyState className="mx-auto mt-5 mb-6" />
```

Comment:
Missed this in the transaction activity update. I've removed the styles associated with these classnames